docs/roadmap.md

```diff
@@ -0,0 +1,49 @@
+# Roadmap to Federated Learning on Azure ML
```

Comment:
Can we replace "roadmap" by "onboarding guide" ? the term Roadmap might give the impression we're talking about the roadmap of the AzureML platform towards FL

Reply:
or "FL ramp-up plan"